src/functionalTest/resources/gradle-example-ci-server-flat/build.gradle

```
@@ -0,0 +1,26 @@
+plugins {
+    id 'java'
+}
+
+repositories {
+    maven {
+        url "should-override-me"
+        credentials {
+            password "should-override-me"
+        }
+    }
+    flatDir {
+        dirs './tests-local-project-dependency/build/libs'
+    }
+}
+
+dependencies {
+    implementation name: 'tests-local-project-dependency'
```

This is illegal syntax in Gradle and might stop working in the future.
Let's add a group and version.
```suggestion
    implementation group: '', name: 'tests-local-project-dependency', version: ''
```